aeat.py

```diff
@@ -1240,6 +1253,10 @@ def calculate_sii(cls):
                     'because is other reports pending to sending')
                 return
             issued_reports = cls.get_issued_sii_reports()
+            for report in issued_reports:
+                if not report.load_date:
+                    report.load_date = date.today() - timedelta(days=config.sii_default_limit_date)
```

Comment:
79 cols
Ensure field is required or add control here, like example I set in domain above.